pyproject.toml

```diff
@@ -116,7 +116,7 @@ pandas = "1.3.4"
 phoenixdb = "0.7"
 pinotdb = ">=0.4.5"
 protobuf = "3.20.2"
-pyathena = ">=1.5.0,<=1.11.5"
+pyathena = "2.25.2"
```

Comment:
pyathena 3.x requires python >= 3.8.1 which conflicts with python >= 3.8 of the redash requirements, so I decided to use 2.x.